templates/csi-templates/csi-cephfs-secret.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
@nehaberry are we going to use this kind of names for yaml files ? I would suggest to use similar to the names in templates/ocs-deployment

Reply:
@ramkiperiy since we have 2 secret files, we would need to make a distinction in names. Also, the ones in templates/ocs-deployment are mostly unique names, hence we could use the same names as mentioned in rook repo. 

Another way could be to have separate directories for rbd and cephfs and then keep the yaml names same as [here](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/example/cephfs/storageclass.yaml
)

Please suggest.

Reply:
I like the idea of creating subdirectories for each ceph backend and have same yaml file's names

Reply:
@RazTamir I see in latest master branch, we now have a hierarchy as : ocs-deployment/csi/rbd and ocs-deployment/csi/cephfs. Do you suggest to use the same folders ?
@vavuthu ^^ @ramkiperiy

Reply:
From my side yes.
But 1 small thing that is wrong imo. those subdirectories shouldn't be under 'ocs-deployment'

Reply:
@RazTamir yeah I agree. For @vavuthu they need to be under ocs-deployment, as he has added the rbac files for csi there. 

But for us, we can have it outside of ocs-deployment. Let me do that and re-send this patch. thanks :)